web/src/components/njwds-extended/HorizontalStepper.tsx

```diff
@@ -40,6 +44,25 @@ export const HorizontalStepper = (props: Props): ReactElement => {
     }
   };
 
+  const determineAriaState = (stepperState: StepperState): string => {
+    switch (stepperState) {
+      case "ERROR":
+        return "error";
+      case "ERROR-ACTIVE":
+        return "error";
+      case "COMPLETE":
+        return "complete";
+      case "COMPLETE-ACTIVE":
+        return "complete";
+      case "INCOMPLETE-ACTIVE":
+        return "incomplete";
+      case "INCOMPLETE":
+        return "incomplete";
+      default:
+        return "";
```

Comment:
[sand] - I think its quite likely this should be in content and will require a follow up.

Not addressed here because scope has already creeped a lot